scripted-tests/run/build-library/build.sbt

```diff
@@ -15,46 +15,52 @@ scalaVersion := {
 
 nativeConfig := {
   val prev = nativeConfig.value
-  val outPath = crossTarget.value / "libtest.so"
+  val filename = if (Platform.isWindows) "test.dll" else "libtest.so"
```

Comment:
You probably already know this, but the name on macs is typically `.dylib`. Now that you're doing something platform-specific, maybe worth adding here too.

Reply:
Yeah, I am working on a PR to internalize those changes inside the tools API - too much of this stuff has leaked into the plugin.